tests/components/sensoterra/conftest.py

```diff
@@ -0,0 +1,24 @@
+"""Common fixtures for the Sensoterra tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from .const import API_TOKEN
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with (
+        patch(
+            "homeassistant.components.sensoterra.async_setup_entry",
+            return_value=True,
+        ),
+        patch(
+            "sensoterra.customerapi.CustomerApi.get_token",
+            return_value=API_TOKEN,
+        ) as mock_setup_entry,
+    ):
+        yield mock_setup_entry
```

Comment:
Please split the mocks

Reply:
Oh and we should patch the CustomerApi object where we use it, so `homeassistant.components.sensoterra.*`

Reply:
I just pushed code which test the `unique_id`. 

> Oh and we should patch the CustomerApi object where we use it, so homeassistant.components.sensoterra.*

I can't patch like: 
```python
@pytest.fixture
def mock_get_token() -> Generator[AsyncMock, None, None]:
    """Override async_setup_entry."""
    with (
        patch(
            "sensoterra.customerapi.*",
            return_value=API_TOKEN,
        ) as mock_entry,
    ):
        yield mock_entry
```
It gives me a ` AttributeError: <module 'sensoterra.customerapi' from '.../customerapi.py'> does not have the attribute '*'`. In the config flow tests we only need `get_token()`, so I guess we're good for now.

I splitted the patches above, but I eventually remove the patch on `homeassistant.components.sensoterra.async_setup_entry`. Tests do not seem to be affected.

Reply:
No I mean, `homeassistant.components.sensoterra.CustomerApi` will patch the usage of `CustomerApi` in `__init__.py`. Check Airgradient for an example